src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -787,6 +787,41 @@ export default defineConfig({
 });
 ```
 
+## Merging remote font families
+
+<p>
+
+<Since v="5.15.6" />
+</p>
+
+Using a single font family, it is not possible to only download font files for weights normal `500`, italic `500` and normal `600`. This is because a font family is defined by a combination of weights and styles, so you'd necessarily have to also download italic `600`.
+For more advanced cases, families that have the same `cssVariable`, `name` and `provider` will be merged. That means you could achieve the desired result like this:
```

Comment:
```suggestion
A font family is defined by a combination of weights and styles (e.g. `weights: [500, 600]`) and `styles: ["normal", "bold"]`), but you may want to download only certain combinations of these.

For greater control over which font files are downloaded, you can specify the same font (`cssVariable`, `name`, and `provider`) multiple times with different combinations and Astro will merge the results and download only the required files. For example, it is possible to download normal `500` and `600` while downloading only italic `500`:

```
So this wording might need some help, but what about something like this? Is that representative of the feature?